debian/rules

```diff
@@ -16,5 +16,7 @@
 %:
 	dh $@
 
+override_dh_auto_clean:
+	rm -f ssat
```

Comment:
is this relevant?

Reply:
Ah it's not relevant to this PR but minor patch to clean files generated by the previous build.